multimodal/src/autogluon/multimodal/optimization/utils.py

```diff
@@ -78,6 +75,7 @@ def get_loss_func(
     problem_type: str,
     mixup_active: bool,
     loss_func_name: Optional[str] = None,
+    loss_config: Optional[DictConfig] = None,
```

Comment:
`loss_config` is not a good name since it contains more than name says. How about just calling it `config` and only assigning it the `predictor._config.optimization`? Also, add a docstring to explain `config`, i.e., the optimization config.